pmda/density.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import MDAnalysis as mda
+import time
+from MDAnalysis.lib.util import fixedwidth_bins
+from MDAnalysis.analysis.density import Density
+from .parallel import ParallelAnalysisBase
+
+class DensityAnalysis(ParallelAnalysisBase):
+    """Parallel density analysis.
+
+    Parameters
+    ----------
+    universe : MDAnalysis.Universe
+            :class:`MDAnalysis.Universe` object with a trajectory
+    atomselection : str (optional)
```

Comment:
This is currently needed for updating atomgroups, which are not properly handled in PMDA yet, see #73

Reply:
We ultimately want to get rid of it and make it work automatically if `atomgroup` is a [dynamic selection](https://www.mdanalysis.org/docs/documentation_pages/selections.html#dynamic-selections), i.e., a [UpdatingAtomGroup](https://www.mdanalysis.org/docs/documentation_pages/core/groups.html?highlight=updatingatomgroup#MDAnalysis.core.groups.UpdatingAtomGroup)